tests/pytests/functional/modules/test_cmdmod.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+import salt.modules.cmdmod as cmdmod
+import salt.utils.path
+
+pytestmark = [
+    pytest.mark.windows_whitelisted
+]
+
+
+@pytest.mark.skip_unless_on_windows(reason="Only run on Windows")
+def test_powershell():
+    """
+    Test cmd.powershell
+    """
+    ret = cmdmod.powershell("Write-Output foo")
+    assert ret == 'foo'
+
+
+@pytest.mark.skip_unless_on_windows(reason="Only run on Windows")
```

Comment:
I suggested a change in the above `pytestmark` so that you don't have to add this decorator to all test functions.

Reply:
Done